FhirToDataLake/docs/FHIR-Data-Mapping.md

```diff
@@ -78,41 +78,46 @@ Example:
 "extension": "[{\"url\": \"http://nema.org/fhir/extensions#0010:1010\",\"valueQuantity\": {\"value\": 56,\"unit\": \"Y\"}}]"
 }
 ```
-   
 
-### Exclude primitive extensions and inline resources
+**Inline Resources**
 
-Extensions on primitive types usually begin with a underline prefix in the raw FHIR JSON data. Inline [Resources](https://www.hl7.org/fhir/resource.html#Resource) such as "contained" and "outcome" properties are flexiable FHIR resource data properties. These two kinds of fields are excluded in Parquet files.
+Inline [Resources](https://www.hl7.org/fhir/resource.html#Resource) such as "contained" and "outcome" properties are flexible FHIR resource data properties, and will be wrapped into a single string.
```

Comment:
Maybe we can add some reference to contained and outcome properties like https://build.fhir.org/references.html#contained

Reply:
Updated.